### PR TITLE
Create oyepriyansh.json

### PR DESCRIPTION
**Pull request by oyepriyansh**
## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [x] My change requires changes to the documentation.
- [x] I have updated the documentation accordingly.
- [x] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/5621"></a>

### CODE REVIEW

File: data/oyepriyansh.json [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+ "group":"Socials", 
+ "name":"GitHub:Collaborateron Open Source", 
+ "url":"https://github.com/oyepriyansh", 
+ "icon":"FaGithub" 
+ }, 
+ { 
+ "group":"Socials", 
+ "name":"LinkedIn: Let's connect", 
+ "url":"https://linkedin.com/in/oyepriyansh", 
+ "icon":"FaLinkedin"
```
Comment (sidharthviz) [inline data/oyepriyansh.json:62]: add comma line 62

File: data/oyepriyansh.json [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+ "name":"LinkedIn: Let's connect", 
+ "url":"https://linkedin.com/in/oyepriyansh", 
+ "icon":"FaLinkedin" 
+ "color":"#0033cc" 
+ }, 
+ { 
+    "group":"Socials", 
+    "name":"Discord: Let`s hang out", 
+    "url":"https://discord.com/invite/AeAjegXn6D", 
+    "icon":"FaDiscord"
```
Comment (sidharthviz) [inline data/oyepriyansh.json:69]: Here also add comma line no. 69

File: data/oyepriyansh.json [hunk 3]
```diff
@@ -0,0 +1,115 @@
...
+    "name":"Discord: Let`s hang out", 
+    "url":"https://discord.com/invite/AeAjegXn6D", 
+    "icon":"FaDiscord" 
+    "color":"#0033cc" 
+    }, 
+{ 
+ "group":"Sponsor", 
+ "name":"Github Sponsor", 
+ "url":"https://github.com/sponsors/oyepriyansh", 
+ "icon":"SiGithubsponsors"
```
Comment (sidharthviz) [inline data/oyepriyansh.json:76]: comma missing  Line no 76, 83, 90, 97, 104, 111 
Please use https://jsonlint.com/   @oyepriyansh  This might help

**Review by sidharthviz — CHANGES_REQUESTED** [review]

File: data/oyepriyansh.json (hunk 3 above)
Comment (oyepriyansh) [reply data/oyepriyansh.json:76]: Sure, thank you!

**Pull request closed without merging**